docs/admin/node-conformance.md

```diff
@@ -0,0 +1,110 @@
+---
+assignees:
+- Random-Liu
+
+---
+
+* TOC
+{:toc}
+
+## Node Conformance Test
+
+*Node conformance test* is a test framework validating whether a node meets the
+minimum requirement of Kubernetes with a set of system verification and
+functionality test. A node which passes the test is qualified to join a
+Kubernetes cluster.
+
+## Limitations
+
+There are following limitations in the current implementation of node
+conformance test. They'll be improved in future version.
+
+* Node conformance test only supports Docker as the container runtime.
+* Node conformance test doesn't validate network related system configurations
+  and functionalities.
+
+## Prerequisite
+
+Node conformance test is used to test whether a node is ready to join a
+Kubernetes cluster, so the prerequisite is the same with a standard Kubernetes
```

Comment:
Are there any documentation on the "prerequisite" of a standard kubernetes npde?